messaging/MessagingExample/AppDelegate.m

```diff
@@ -125,6 +125,10 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
 
   // Print full message.
   NSLog(@"%@", userInfo);
+
+  if (completionHandler) {
```

Comment:
Same here, as the documentation states: "the completion handler _must_ be called", so it wouldn't make sense to check it.